R/shape_measures.R

```diff
@@ -25,46 +25,88 @@
 GetShape <- function (pash, type = "all") {
   TestClass(pash)
   lt = pash[["lt"]]
-  if (identical(type, "LTentr")) S = c(LTentr = LifetableEntropy(lt$nax, lt$nx, lt$ndx, lt$ex))
-  if (identical(type, "LTgini")) S = c(LTgini = LifetableGini(lt$nx, lt$nax, lt$lx, lt$ex))
-  if (identical(type, "LTcv"))   S = c(LTcv = LifetableCV(lt$nx, lt$nax, lt$dx, lt$ex))
+  if (identical(type, "LTentr"))
```

Comment:
Please add brackets for the if conditions. Me leaving them out the first time was no good style either.